site/js/misc.js

```diff
@@ -119,7 +119,7 @@ exports.queryItemsAlasql = (query, items) => {
 exports.queryItems = (query, items, exactWord) => {
     query = query.trim().replace(",", ".").toLowerCase();
     if (query.length < 3) return { items: [], queryTokens: [] };
-    const regex = /([\p{L}&-\.][\p{L}\p{N}&-\.]*)|(>=|<=|=|>|<)|(\d+(\.\d+)?)/gu;
+    const regex = /([\p{L}&-\.][\p{L}\p{N}&-\.]*)|(>=|<=|=|>|<)|(\d+[^\s.]+)|(\d+(\.\d+[^\s.]+)?)/gu;
```

Comment:
číslaBezTečkyText (90g)
čísla.číslaText (1.0%)

Reply:
Můžeš sem (případne i do description commitu) přidat příklad textu který se snažíš tím novým regexpem matchovat?

Reply:
Je to tady přece hned 1. comment ??

Reply:
Myslel jsem konkrétní text: např. "mýdlo(90g)" "12.1 kusů mýdla"

Reply:
Tak problematické:
Protex Charcoal tuhé mýdlo s přirozenou antibakteriální ochranou 90g
- totálně rozbité po:
s t r o n g
- a číslo s písmeny na konci taky např. (bez písmen ok, s písmeny nezvýrazní celé slovo):
90g
1.0%